app/src/main/java/org/simple/clinic/home/overdue/removepatient/RemoveAppointmentEvents.kt

```diff
@@ -18,6 +18,6 @@ data class PatientDeadClicked(val patientUuid: UUID) : UiEvent {
   override val analyticsName = "Remove Appointment with Reason:Reason changed to Patient Dead"
 }
 
-data class CancelReasonClicked(val selectedReason: Appointment.CancelReason) : UiEvent {
-  override val analyticsName = "Remove Appointment with Reason:Reason changed to $selectedReason"
+data class CancelReasonClicked(val selectedReason: AppointmentCancelReason) : UiEvent {
+  override val analyticsName = "Remove Appointment with Reason:Reason changed to ${AppointmentCancelReason.TypeAdapter.fromEnum(selectedReason)}"
```

Comment:
Because cancelation reasons are now Kotlin objects, their string representation will not be user readable anymore. E.g., `org.simple.clinic.overdue.AppointmentCancelReason$Moved@6aceb1a5`

Using the type adapter is necessary here, but it makes this string very long. I hope that's okay.

Reply:
What do you feel about adding a property `analyticsName` in the sealed class itself?

```kotlin
sealed class AppointmentCancelReason {
  
  val analticsName: String
    get() = TypeAdapter.fromEnum(this)!!
}
```